src/index.vue

```diff
@@ -122,11 +122,14 @@
                 this.dropzone.emit("addedfile", file);
                 this.dropzone.emit("thumbnail", file, fileUrl);
                 this.dropzone.createThumbnailFromUrl(file, fileUrl, callback, crossOrigin);
-                this.dropzone.emit("complete", file);
-                this.$emit('vdropzone-file-added-manually', file);
+                this.dropzone.emit("complete", file);                
                 if ((typeof options.dontSubstractMaxFiles == 'undefined') || !options.dontSubstractMaxFiles) {
```

Comment:
Will work as worked before :) There is 'or' operation in here. So all old code that is using this library will be unaffected but for new code if you add `dontSubstractMaxFiles : true` in options. Will work in new way. I'm unsure if this is a good way to solve both problem and legacy compatibility but I haven't came up with anything better.